index.html

```diff
@@ -69,5 +70,185 @@ <h2 class="text-2">
         
     </section>
 
+    <section class="wrapper-works">
+        <header class="title">
+            My Recent Works
+        </header>
```

Comment:
- [ ] **[OPTIONAL]:** I'm sorry the last reviewer didn't point this out, but please consider changing this `<header>` tag for an actual `<hx>` tag (where x represents a number different than 1). This is because `<header>` is a special tag reserved for the header of the page which you already defined with the toolbar. I strongly suggest you change this! 🤓 🛠️